cekit/templates/template.jinja

```diff
@@ -74,7 +74,7 @@ rm{% for repo in repositories %} /etc/yum.repos.d/{{ repo.filename }}{% endfor %
  #}
 {%- macro repo_clear_cache(pkg_manager) -%}
     {% if pkg_manager in helper.SUPPORTED_PACKAGE_MANAGERS %}
-rm -rf "/var/cache/yum" "/var/lib/rpm" "/var/lib/dnf" "/var/cache/apt" "/var/cache/dnf"
+rm -rf "/var/cache/yum" "/var/lib/dnf" "/var/cache/apt" "/var/cache/dnf"
```

Comment:
If we're not removing the var/lib/rpm do we need the microdnf clean ? Will that shrink the DB ?

Reply:
TBD -- I need to dig a bit deeper.

Reply:
No - in fact running it makes the image fractionally larger. The `rm`s do the equivalent job.